chapters/pre-recorded-stt/speaker-diarization.mdx

```diff
@@ -59,7 +62,7 @@ Speakers will be assigned indexes by **order of appearance** (i.e. the 1st speak
   }
 }
 ```
-```json diarization_enhanced
+```json enhanced diarization
 {
   "transcription": {...},
   "diarization_enhanced": {
```

Comment:
this key `diarization_enhanced` has been kept for backward-compatibility only but it should be removed from the doc (remove the whole block from l65 to l87)